src/components/about-us/section-01/index.js

```diff
@@ -0,0 +1,247 @@
+import { marginBetweenSections } from '../constants/styles'
+import { screen } from '../utils/screen'
+import LottieAnim from './lottie-animation'
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import titleImg from '../../../../static/asset/about-us/title-section1.png'
+import titleImgMob from '../../../../static/asset/about-us/title-section1-mob.png'
+
+const Container = styled.div`
+  position: relative;
+  margin: 0 auto;
+  background-color: white;
+  overflow: hidden;
+  border: solid 8px #c71b0c;
```

Comment:
Since the primary color is Liver Red `#C7000A` which differs from original primary color according to the color scheme in our new design guideline. I will define a new color name called `liverRed` in `src/themes/common-variables.js`.